sw/device/tests/pwrmgr_smoketest.c

```diff
@@ -58,9 +58,8 @@ bool test_main(void) {
         mmio_region_from_addr(TOP_EARLGREY_AON_TIMER_AON_BASE_ADDR),
         &aon_timer));
     aon_timer_testutils_wakeup_config(&aon_timer, wakeup_threshold);
-    pwrmgr_testutils_enable_low_power(
-        &pwrmgr, kDifPwrmgrWakeupRequestSourceFive,
-        kDifPwrmgrDomainOptionUsbClockInActivePower);
```

Comment:
Should we randomize this at top level to ensure low power entry works regardless of what value we set for this bit?